docs/Makefile

```diff
@@ -175,3 +175,12 @@ pseudoxml:
 	$(SPHINXBUILD) -b pseudoxml $(ALLSPHINXOPTS) $(BUILDDIR)/pseudoxml
 	@echo
 	@echo "Build finished. The pseudo-XML files are in $(BUILDDIR)/pseudoxml."
+
+makedocsmessages: gettext
+	cd docs && sphinx-intl update -p _build/locale -l en
+
+ensurecrowdinclient:
+	ls -l crowdin-cli.jar || wget https://crowdin.com/downloads/crowdin-cli.jar # make sure we have the official crowdin cli client
+
+uploaddocsmessages: ensurecrowdinclient
+	java -jar crowdin-cli.jar upload sources -b `git symbolic-ref HEAD | xargs basename`
```

Comment:
Yes, the `downloadmessages` command would at the very least have to accept an argument. I really like my own proposal to patch up the Python crowdin-client so we can maintain the branch pointers in a `.crowdin.yml`